src/platform/common/utils/localize.ts

```diff
@@ -1396,6 +1396,18 @@ export namespace DataScience {
         );
     export const listOfFilesWithLinksThatMightNeedToBeRenamed = () =>
         localize('DataScience.listOfFilesWithLinksThatMightNeedToBeRenamed', 'File(s): {0} might need to be renamed.');
+    export const noActiveKernelSession = () =>
+        localize('DataScience.noActiveKernelSession', 'No no active kernel session.');
```

Comment:
I don't see this in package.nls.json?
Also not sure this needs to be localized, but not going to ask this to be removed, as this is an error case that should not occur.

Reply:
Because when users recieve this error message, i'm 99% certain that they will not know what to do at that point in time, as the error message doesn't tell them anything apart from the fact that sometjhing is missing.

Reply:
I'll de-localize it!